app/assets/stylesheets/base/_base.scss

```diff
@@ -0,0 +1,2 @@
+@import "./_forms.scss";
```

Comment:
I think you don't to explicitly use `./` here.

Reply:
👌